Source/WebCore/animation/CSSNumberishTime.cpp

```diff
@@ -145,7 +185,10 @@ bool CSSNumberishTime::operator>=(const CSSNumberishTime& other) const
 
 bool CSSNumberishTime::operator==(const CSSNumberishTime& other) const
 {
-    return m_type == other.m_type && m_value == other.m_value;
+    ASSERT(m_type == other.m_type);
+    if (m_type == Type::Time)
+        return std::abs(time()->microseconds() - other.time()->microseconds()) < timeEpsilon.microseconds();
+    return m_value == other.m_value;
 }
```

Comment:
Nit: I generally don't like approximate equal checks to be done inside `operator==` because it's not obvious from the use of `==` that it's doing approximate checks.